test/Flick/AnimatedCarousalPage.xaml.cs

```diff
@@ -126,5 +253,48 @@ private void Grid_KeyDown(object sender, KeyRoutedEventArgs e)
                 Frame.GoBack();
             }
         }
+
+        private void Sv_Tapped(object sender, TappedRoutedEventArgs e)
+        {
+            if (!isScrolling)
+            {
+                // Get x position of the scrollviewer's center point if the currently selected item.Offset.X + item.Size.X/2 was the center point
+                var svCenterPoint = sv.HorizontalOffset + sv.ViewportWidth / 2;
+                svCenterPoint -= (svCenterPoint + layout.Spacing / 2) % (layout.Spacing + layout.ItemWidth);
+                svCenterPoint += layout.Spacing / 2 + layout.ItemWidth / 2;
+                var tapPositionInSV = e.GetPosition(sv).X + sv.HorizontalOffset;
```

Comment:
>X [](start = 56, length = 1)

At some point we will need to support both horizontal and vertical orientations of this panel, so we should be able to handle in x or y depending the orientation. I can show you a trick we use to do it independent of the orientation.